dborutils/batch_manager.py

```diff
@@ -1,130 +0,0 @@
-from datetime import datetime
-from decimal import Decimal
-import logging
-logger = logging.getLogger(__name__)
-
-
-class BatchManager(object):
```

Comment:
Uh, how did this get dropped?  This class is still referred to down in copy_service.py below.